heroku_connect/management/commands/makemappings.py

```diff
@@ -0,0 +1,41 @@
+import json
+
+from django.core.management import BaseCommand
+
+from heroku_connect.utils import get_mapping
+
+
+class Command(BaseCommand):
+    """
+    Return Heroku Connect mapping JSON.
+
+    Example::
+
+        python manage.py makemappings -o hc_mappings.json
+        heroku connect:import hc_mappings.json
+
+    Note:
+        We for the example to work you will need the
+        `Heroku Connect CLI Plugin`_.
+
+    .. _`Heroku Connect CLI Plugin`: https://github.com/heroku/heroku-connect-plugin
+
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('-o', metavar='file_name', dest='output',
+                            help='Output file name.')
+
+    def handle(self, *args, **options):
+        output = options.get('output', None)
+        if output:
+            f = open(output, 'w+')
+        else:
+            f = self.stdout
+
+        mapping = get_mapping()
+
+        try:
+            json.dump(mapping, f)
+        finally:
+            f.close()
```

Comment:
It might be `sys.stdout` getting closed here. Should we really? Not sure who might still try to write to that handle...

Reply:
You are right, given this is a finally statement that wraps the entire code, I guess no one. I don't know. If they don't want this to be closed, they should not expose the API? It certainly is implemented.
I don't want to make this more complex and an open file needs to be closed. What do you think? Make the code more complex or ignore an unlikely side effect?

Reply:
I hope my answer doesn't sound suggestive. I really want your honest opinion ;)